dev/com.ibm.ws.microprofile.faulttolerance/src/com/ibm/ws/microprofile/faulttolerance/impl/async/QueuedFuture.java

```diff
@@ -218,44 +206,78 @@ public R get(long methodTimeout, TimeUnit methodUnit) throws InterruptedExceptio
      * @throws Exception
      */
     @Override
-    public Future<R> call() throws Exception {
+    @FFDCIgnore(Throwable.class)
+    public Void call() throws Exception {
```

Comment:
This class can be a Runnable rather than a Callable

Reply:
Done